tools/update_version.sh

```diff
@@ -0,0 +1,140 @@
+    log_info "Valid <plugin name> choices are:
+$all_plugins"
+    log_info "<plugin version> $tab-> Specify the new version of the given plugin"
+
+} 2> /dev/null
+
+package=
+version=
+dev=false
+while (( $# > 0 )); do
+    case "$1" in
+        --debug) ;&
+        -d) set -o xtrace ;;
+        --dev) dev=true ;;
+        --help) ;&
+        -h) usage ; exit 0 ;;
+        *)
+            if [ -z "$package" ]; then
+                if grep -q "$1" <<< "$all_plugins"; then
+                    package="$1"
+                else
+                    {
+                        log_error "Illegal argument $1. Expected one of:
+$all_plugins"
+                    } 2>/dev/null
+                    exit 93
+                fi
+            elif [ -z "$version" ]; then
+                version="${1/v/}"
+            else
+                {
+                    log_error "Illegal argument $1. Already saw package '$package' and version '$version'"
+                    log_error "This script expects two and only two non -flag arguments, <package name> and <package version>"
+                } 2>/dev/null
+                exit 94
+            fi
+       ;;
+    esac
+    shift
+done
+
+if [ -z "$package" ]; then
+    {
+        log_error "Expected exactly two arguments <package name> <package version>"
+        log_error "Valid choices for <package name> are:
+$all_plugins"
+    } 2>/dev/null
+    exit 92
+fi
+if [ -z "$version" ]; then
+    {
+        log_error "Did not receive a second argument of a version to set $package to"
+    } 2>/dev/null
+    exit 91
+fi
+
+function update_file() {
+    local file="$1"
+    local prefix="$2"
+    local suffix="$3"
+    local extra="${4:-}"
+    sed -i "s/${prefix}.*/${prefix}${version}${extra}${suffix}/g" "$ROOT_DIR/plugins/$file"
+    git add "$ROOT_DIR/plugins/$file"
+    git commit -m "chore(version): update $package to version $version${extra}"
+}
+
+extra=
+[ "$dev" = true ] && extra=".dev0"
+case "$package" in
+        auth-keycloak)
+            update_file auth-keycloak/src/js/package.json '"version": "' '",'
+            ;;
+        dashboard-object-viewer)
+            update_file dashboard-object-viewer/src/js/package.json '"version": "' '",'
+            ;;
+        json)
+            update_file json/src/deephaven/plugin/json/__init__.py '__version__ = "' '"' "$extra"
+            ;;
+        matplotlib)
+            update_file matplotlib/setup.cfg 'version = ' '' "$extra"
+            ;;
+        plotly)
+            update_file plotly/src/deephaven/plugin/plotly/__init__.py '__version__ = "' '"' "$extra"
+            ;;
+        plotly-express)
+            update_file plotly-express/setup.cfg 'version = ' '' "$extra"
+            ;;
+        table-example)
+            update_file table-example/src/js/package.json '"version": "' '",'
+            ;;
+        ui)
+            update_file ui/src/deephaven/ui/__init__.py '__version__ = "' '"' "$extra"
```

Comment:
I added a ticket for harmonizing the version info: https://github.com/deephaven/deephaven-plugins/issues/212

I know I also said not to worry about JS packages here ... but it looks like it would be very easy to add them as well (even though we expect them to only be published in the short term), e.g.
```suggestion
            update_file ui/src/deephaven/ui/src/js/package.json '"version": "' '",'
            update_file ui/src/deephaven/ui/__init__.py '__version__ = "' '"' "$extra"
```